plugin/pkg/scheduler/algorithm/BUILD

```diff
@@ -21,6 +21,7 @@ go_library(
         "//pkg/api/v1:go_default_library",
         "//pkg/apis/extensions/v1beta1:go_default_library",
         "//plugin/pkg/scheduler/api:go_default_library",
+        "//vendor:k8s.io/client-go/pkg/apis/apps",
```

Comment:
Thanks! I thought you were referring to k8s.io/kubernetes/pkg/apis/extensions/v1beta1. This is where ReplicaSet is coming from in Scheduler's code, but does not have StatefulSet. I am going to use k8s.io/kubernetes/pkg/apis/apps/v1beta1.